cmd/livepeer_cli/livepeer_cli.go

```diff
@@ -58,7 +59,7 @@ func main() {
 		return nil
 	}
 	app.Version = core.LivepeerVersion
-	// flag.Parse()
+	flag.Parse()
```

Comment:
I think that we cannot parse flags here, because they need to be parsed in [Urfave CLI framework](https://github.com/urfave/cli).

I think after your change, the flags won't work. Please check if the following works for you.

```
./livepeer_cli -http 7937
```

Reply:
Yep, you're right. Silly of me to not realize this before opening up for review - will revisit the CLI framework and fix this PR.